src/SB.Abstraction/Service/ServiceBusClient.cs

```diff
@@ -35,15 +35,15 @@ private void LoadConfig()
         private IClient CreateClients(dynamic option)
         {
             IClient busClient;
-            if (option._type == PUBLISHER) busClient = new PublisherClient(client);
-            else busClient = new ListenerClient(client);
+            if (option._type == PUBLISHER) busClient = new PublisherClient(client, option._nameQueue, config.Subscription);
```

Comment:
consider define PUBLISHER and LISTENER as a enum

Reply:
could be too. i created them like CONSTS